.travis.yml

```
@@ -31,6 +31,8 @@ matrix:
     env: GOFLAGS=-mod=vendor GO111MODULE=on
   - go: "1.13.x"
     env: GOFLAGS=-mod=vendor
+  - go: "1.14.x"
+    env: GOFLAGS=-mod=vendor
   - go: "tip"
     env: GOFLAGS=-mod=vendor
   - go: "1.13.x"
```

```suggestion
  - go: "1.14.x"
```

and same for `GOARCH=amd64`

---

> same for GOARCH=amd64